src/drivers/BoostDriver.cpp

```diff
@@ -60,8 +60,8 @@ class CukeBoostLogInterceptor : public ::boost::unit_test::unit_test_log_formatt
     void log_exception_finish( std::ostream& ) {};
 
     void log_entry_start( std::ostream&, log_entry_data const&, log_entry_types /*let*/) {};
-    void log_entry_value( std::ostream&, const_string /*value*/);
-    void log_entry_value( std::ostream&, lazy_ostream const& /*value*/) {};
+    void log_entry_value( std::ostream&, const_string value);
+    void log_entry_value( std::ostream&, lazy_ostream const& value);
```

Comment:
Please don't do this: this causes warnings about "unused parameters", that's the reason it was commented out like that in the first place.